neal/sampler.py

```diff
@@ -289,46 +350,78 @@ def _default_ising_beta_range(h, J):
 
     Assume each variable in J is also in h.
 
-    We use the minimum bias to give a lower bound on the minimum energy gap, such at the
-    final sweeps we are highly likely to settle into the current valley.
+    Generally one should optimize min_beta, max_beta and the shape of the schedule (linear, 
+    geometric) to maximize some objective such as time to solution. 
+    The approximations here are simplified, and can be far from optimal in some cases, use
+    custom schedules to address such cases. 
+    Approximations used are of complexity O(number of biases), O(1) approximations are also 
+    possible, as are stronger approximations, but methods are not bottlenecks in practice.
+    We use the maximum bias per spin to give an upper bound on the gap, allowing the initial 
+    sweeps to be fast mixing.
+    We use an approximation to the minimum bias per spin to give a lower bound on the minimum 
+    energy gap, such that at the final sweeps we are highly unlikely to excite away from a 
+    global (or local) minima.
```

Comment:
Nitpick: "a global (or local) minima" mixes singular and plural. I think you wat "minimum"